README.md

```diff
@@ -73,6 +97,8 @@ Usage
           bindkey "$terminfo[kcuu1]" history-substring-search-up
           bindkey "$terminfo[kcud1]" history-substring-search-down
 
+      Users have also observed that `[OA` and `[OB` are correct values, _even if_ these were not the observed values.
```

Comment:
Please wrap this long line to 80 characters like the rest of the README (except for command/output listings, of course).